packages/cardano-services/src/TxSubmit/openApi.json

```diff
@@ -6,13 +6,13 @@
       "name": "Apache 2.0",
       "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
     },
-    "version": "1.0.0"
+    "version": "2.0.0"
```

Comment:
What's the breaking change in this API?

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/769/files#diff-2965a6e837d0d4d19bab9dba952b3a13ca71a7874430303e95a2cf1dad4b863bR9